src/FileType.php

```diff
@@ -122,6 +122,7 @@ class FileType extends Enum
             Extension::OGG,
             Extension::WAV,
             Extension::MIDI,
+            Extension::M4A,
```

Comment:
tohle je celkem hra na kocku a na mys, nemeli bychom tam mit nejaky default fallback a hlidat warningy? Nebo proc to resime?